docs/img/example.cast

```diff
@@ -0,0 +1,282 @@
+{"width": 104, "timestamp": 1541016888, "env": {"TERM": "xterm-256color", "SHELL": "/usr/bin/zsh"}, "height": 25, "version": 2}
```

Comment:
What is this file?
Is it used somewhere?

Reply:
It's the original "asciicast" file that I've included for historical purposes / if we need to do any further modifying to it.